test/QuarkBuilder.t.sol

```diff
@@ -315,7 +315,7 @@ contract QuarkBuilderTest is Test {
     function testSimpleBridgeTransferWithPaycallSucceeds() public {
         QuarkBuilder builder = new QuarkBuilder();
         PaymentInfo.PaymentMaxCost[] memory maxCosts = new PaymentInfo.PaymentMaxCost[](2);
-        maxCosts[0] = PaymentInfo.PaymentMaxCost({chainId: 1, amount: 1e5});
+        maxCosts[0] = PaymentInfo.PaymentMaxCost({chainId: 1, amount: 5e5});
         maxCosts[1] = PaymentInfo.PaymentMaxCost({chainId: 8453, amount: 1e5});
```

Comment:
Using `e5` is kind of confusing. These should be 0.5e6 and 0.1e6 right? Or just make them 5e6 and 1e6.